src/cdk-experimental/config.bzl

```diff
@@ -0,0 +1,13 @@
+entryPointGroups = [
+    ["clipboard"],
+    ["dialog"],
+    ["popover-edit"],
+    ["scrolling"],
+]
+
+# List of all entry-points of the Angular cdk-experimental package.
+CDK_EXPERIMENTAL_ENTRYPOINTS = [ep for group in entryPointGroups for ep in group]
+
+# List of all entry-point targets of the Angular cdk-experimental package.
+CDK_EXPERIMENTAL_TARGETS = ["//src/cdk-experimental"] + \
```

Comment:
Are the `\`s at the end of line necessary?

Reply:
Yes. Otherwise buildifier will just leave them in the same line which makes the whole `config.bzl` file less readable.